Update securing_applications guide for latest adapter changes (community) (#20995)
keycloak/keycloak: docs/documentation/securing_apps/topics/saml/mod-auth-mellon.adoc

=== mod_auth_mellon Apache HTTPD Module

> WARNING: {project_name} does not provide any official support to mod_auth_mellon. The instructions below are best-effort and may not be up-to-date.
> It is recommended to stick to official mod_auth_mellon documentation for more details.

Reviewer: ```suggestion
WARNING: {project_name} does not provide any official support to mod_auth_mellon. The instructions below are best-effort and may not be up-to-date.
We recommend that you stick to official mod_auth_mellon documentation for more details.
```
